README.md

```diff
@@ -1 +1,17 @@
-# go-option
\ No newline at end of file
+<a href="http://tarantool.org">
+	<img src="https://avatars2.githubusercontent.com/u/2344919?v=2&s=250" align="right">
+</a>
+
+[![Go Reference][godoc-badge]][godoc-url]
+[![Actions Status][actions-badge]][actions-url]
+[![Code Coverage][coverage-badge]][coverage-url]
+[![Telegram][telegram-badge]][telegram-url]
+
+[godoc-badge]: https://pkg.go.dev/badge/github.com/tarantool/go-option.svg
+[godoc-url]: https://pkg.go.dev/github.com/tarantool/go-option
+[actions-badge]: https://github.com/tarantool/go-option/actions/workflows/testing.yml/badge.svg
+[actions-url]: https://github.com/tarantool/go-option/actions/workflows/testing.yml
+[coverage-badge]: https://coveralls.io/repos/github/tarantool/go-option/badge.svg?branch=master
+[coverage-url]: https://coveralls.io/github/tarantool/go-option?branch=master
+[telegram-badge]: https://img.shields.io/badge/Telegram-join%20chat-blue.svg
+[telegram-url]: http://telegram.me/tarantool
```

Comment:
Please, add a link to a russian chat too.

Reply:
fixed